Sources/TensorFlow/Operators/Math.swift

```diff
@@ -686,6 +686,24 @@ func _vjpRelu<T: TensorFlowFloatingPoint>(
     return (relu(x), { v in Tensor(x .> 0) * v })
 }
 
+/// Checks which elements of `x` are finite.
+@inlinable
+public func isFinite<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<Bool> {
```

Comment:
I think they should be defined as computed properties on `Tensor`. The names sound good as properties.

Reply:
Sounds good. Done in the last commit.